packages/@react-aria/overlays/test/calculatePosition.test.ts

```diff
@@ -231,7 +234,7 @@ describe('calculatePosition', function () {
       offsetAfter: [350, 300, 196, undefined, 450],
       crossAxisOffsetPositive: [210, 50, 196, undefined, 200],
       crossAxisOffsetNegative: [190, 50, 196, undefined, 200],
-      mainAxisOffset: [200, 40, 196, undefined, 200],
+      mainAxisOffset: [200, 40, 196, undefined, 190],
```

Comment:
This seemed like a bug, max height didn't change if the `top` positioned overlay was pushed upwards via `offset`